src/pages/studyView/StudyViewConfig.ts

```diff
@@ -185,6 +189,14 @@ const studyViewFrontEnd = {
                 w: 2,
                 h: 2,
             },
+            [ChartTypeEnum.SAMPLE_TREATMENTS_TABLE]: {
+                w: 2,
+                h: 2,
+            },
+            [ChartTypeEnum.PATIENT_TREATMENTS_TABLE]: {
```

Comment:
whats going on here... these are all the same key 'table'

Reply:
How so? `ChartTypeEnum.PATIENT_TREATMENTS_TABLE` evaluates to `PATIENT_TREATMENTS_TABLE`

Reply:
oh sorry, I got confused by ChartTypeNameEnum above